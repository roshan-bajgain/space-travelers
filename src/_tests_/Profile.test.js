import React from "react";
import renderer from "react-test-renderer";
import { render, screen } from "@testing-library/react";
import { Provider } from "react-redux";
import store from "../redux/configureStore";
import Profiles from "../components/Space-profiles";

describe("Mission Element Tests", () => {
  test("Check if Component Renders in DOM", () => {
    render(
      <Provider store={store}>
        <Profiles />
      </Provider>
    );
    expect(screen.queryByText(/My Rockets/)).toBeTruthy();
    expect(screen.queryByText(/My Missions/)).toBeTruthy();
  });
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Profiles />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
