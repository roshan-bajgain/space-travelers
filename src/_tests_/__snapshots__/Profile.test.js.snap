// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission Element Tests renders correctly 1`] = `
<div
  className="grid grid-cols-2 w-full mx-auto"
>
  <section>
    <h2
      className="text-2xl w-full flex justify-center items-center p-3 font-bold"
    >
      My Rockets
       
    </h2>
     
    <ul
      className=" flex flex-col px-6 py-3"
    >
       
       
    </ul>
     
  </section>
   
  <section>
    <h2
      className="text-2xl w-full flex justify-center items-center p-3 font-bold"
    >
      My Missions
       
    </h2>
     
    <ul
      className=" flex flex-col px-6 py-3"
    >
       
       
    </ul>
     
  </section>
   
</div>
`;
